modules/cluster-logging-maintenance-support-list.adoc

```diff
@@ -7,19 +7,14 @@
 
 You must set the Red{nbsp}Hat OpenShift Logging Operator to the `Unmanaged` state to modify the following components:
 
-* The `Elasticsearch` custom resource (CR)
-
-* The Kibana deployment
-
 * The `fluent.conf` file
```

Comment:
If this doc is for 6.x.  fluentd related section should be removed.

Reply:
this is for 5.x